### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by drxeno02**
### What Does This PR Do?
Update CHANGELOG.md

### Where Should The Reviewer Start?
CHANGELOG.md

### How Should This Be Manually Tested?
N/A

### Background Context
N/A

### Screenshots
N/A

### Checklist (if applicable):
[ ] item 1
[ ] item 2
[ ] item 3
[ ] item 4
[ ] item 5

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,14 +5,32 @@ All notable changes to this project will be documented in this file.
...
+<a name="1.1.0"></a>
+## [1.1.0] - 2021-12-17
+
+### Platform Compatibility
+
+- Compatibile for both iOS and Android platforms.
+
+### Added Interfaces
+
+- TrackHandledException(): Use this to log any handled exceptions. This should be added to try-catch exceptions
```
Comment (Debadri-Redapple) [inline CHANGELOG.md:30]: A change needed here.
```
TrackHandledExceptionRequest.Builder()
    .SetException(exception)
    .Create(
    trackHandledExceptionRequest => MikrosManager.Instance.AnalyticsController.LogEvent(trackHandledExceptionRequest),
    onFailure =>
    {
        // handle failure
    });
```

Comment (drxeno02) [reply CHANGELOG.md:30]: @Debadri-Redapple This is a CHANGELOG, not a README with examples for how to use the function. This example goes into the MIKROS SDK README, and also this repo README (which isn't updated yet).

Comment (drxeno02) [reply CHANGELOG.md]: @Debadri-Redapple I did update to the CHANGELOG to represent that it is a request --> `TrackHandledExceptionRequest()`. That should be more clear.

**Review by Debadri-Redapple — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Debadri-Redapple) [reply CHANGELOG.md]: Yes, that's what I was implying.

**Pull request merged**